index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Cereal</title>
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+	<article class="holec">
+		<header> 
+			<h1>Honey Bunches of Oats with Almonds</h1>
+		</header>
+		<sectiona>
+			<section class="firstsec"><img src="honeybunches.jpg" width= "350" height="350"></section>
+
+			<section class="secondsec">In 1986, Post employee Vernon J. Herzing began mixing together various Post cereals to create new ones, with his daughter Kimberly assisting as the official taste-tester. And it was on a Saturday in 1989 when Kimberly took a bite of her father’s latest creation that Vern knew he had something special. That was the day Vern created Honey Bunches of Oats.
+			</section>
+
+			<section class="thirdsec">Available Now!</section>
+		</sectiona>
+
+		<sectionb>
+			<h4>Specifications</h4>
+			 <p class="explparg">Size 20 x 30</p>
+			 <p class="explparg">Weight: 27oz</p>
+		    <footer class="specfut">Specifications valid until 08/02/2016</footer>
+		</sectionb>
+
+		<sectionc>
+			<h4 id="spcup">Pricing</h4>
+			 <p class="explparg">1: $2.70</p>
+			 <p class="explparg">2-50: $2.0</p>
+		     <p class="explparg">51+: $1.7</p>
+		</sectionc>
+	</article>
+
+	
+
+	<article class="holec">
+		<header> 
+			<h1 class="onel">Cocoa Krispies</h1>
+		</header>
+		<sectiona>
+			<section class="firstsec"><img src="cocoakrispies.jpeg" width= "350" height="350"></section>
+
+			<section class="secondsec">Cocoa Krispies, Choco Krispis, Choco Krispies, Coco Pops, or Choco Pops is a breakfast cereal produced by Kellogg's, coming both as a boxed cereal and as a snack bar with a 'dried milk' covered bottom to make the cereal-with-milk tradition portable. It is a cocoa-flavored version of Rice Krispies. Containing a substance imitating milk chocolate.
+			</section>
+
+			<section class="thirdsec">Available Now!</section>
+		</sectiona>
+
+		<sectionb>
+			<h4>Specifications</h4>
+			 <p class="explparg">Size 20 x 30</p>
+			 <p class="explparg">Weight: 27oz</p>
+		    <footer class="specfut">Specifications valid until 08/02/2016</footer>
+		</sectionb>
+
+		<sectionc>
+			<h4 id="spcup">Pricing</h4>
```

Comment:
ids must be unique. But since you are only using this for css, change this to a class.